source/renderer/app/components/wallet/WalletRestoreDialog.scss

```diff
@@ -108,9 +100,16 @@
           }
         }
 
+        .tooltip {
+          height: 12px;
+          left: -147px;
+          position: relative;
+          top: 2px;
+          width: 12px;
```

Comment:
Same here, you can check https://atomiks.github.io/tippyjs/v6/all-props for available props that might be used instead

Reply:
Fixed